js/weekView.js

```diff
@@ -1,7 +1,204 @@
-import { isSameDate, isSameWeek, getWeekDates, selectedDate, getToday, HOUR_COUNT } from "../js/dateManipulation.js"
+import { isSameDate, isSameWeek, getWeekDates, getToday, HOUR_COUNT, LOCALE } from "../js/dateManipulation.js"
+import {selectedDate, storageState} from './state.js'
+import {findByHour} from './database.js'
+import { displayModal } from "./event.js"
+
+const EVENTBUBBLE_OFFSET = 25
+
+const sortByKey = (objectArray, key) => {
+    return objectArray.sort( (a,b) => {
+        if(a[key] < b[key]){
+            return 1
+        }
+        if(b[key] < a[key]){
+            return -1
+        }
+        return 0
+
+    })
+}
+
+const createEventCard = (eventId, targetElem) => {
+
+    const event = JSON.parse(localStorage.getItem(eventId))
+
+    const container = document.createElement('div')
+    container.classList = "container eventCard"
+
+    const DOMrect = targetElem.getBoundingClientRect()
```

Comment:
I would suggest all variables to start in lowerCase. Even github highlights upper-case variables differently because it thinks it is constant or class.